### PR TITLE
CAKE-5209

### PR DESCRIPTION
**Pull request by vforge**
## Links

* http://wikia-inc.atlassian.net/browse/CAKE-5209

## Description

- Re-work how post-search-results work, introduce post-search-interrupt that wraps some of the logic
- Add post-search-results to article content

## Reviewers

@Wikia/cake

### CODE REVIEW

File: app/components/article-content.js [hunk 1]
```diff
@@ -44,9 +44,6 @@ export default Component.extend(
     lang: reads('wikiVariables.language.content'),
     dir: reads('wikiVariables.language.contentDir'),
     isFastBoot: reads('fastboot.isFastBoot'),
-    affiliateUnit: computed('displayTitle', function () {
```
Comment (JoshuaRogan) [inline app/components/article-content.js:47]: `  2:10  error  'computed' is defined but never used  no-unused-vars`

**Review by JoshuaRogan — APPROVED** [review]
🚢

File: app/components/article-content.js (hunk 1 above)
Comment (vforge) [reply app/components/article-content.js:47]: 5b1746d

File: app/components/article-content.js [hunk 2]
```diff
@@ -619,7 +619,33 @@ export default Component.extend(
...
+        }));
+      }
+    },
+
+    /**
+     * Injects a post search results into the article content
+     * @param {string} title
+     */
+    handlePostSearchResults(title) {
+      // search for second section
```
Comment (mklucsarits) [inline app/components/article-content.js]: fourth section ;)

Comment (vforge) [reply app/components/article-content.js]: Bah! ;)

Comment (vforge) [reply app/components/article-content.js]: 0b13d9f

**Review by mklucsarits — APPROVED** [review]

**Pull request merged**